README.md

```diff
@@ -213,7 +217,6 @@ is also supported:
 ```
 
 [datetime.date]: https://docs.python.org/3/library/datetime.html#available-types
-[header]: https://dicom-parser.readthedocs.io/en/latest/modules/dicom_parser.html#dicom_parser.header.Header
+[Header]: https://dicom-parser.readthedocs.io/en/latest/modules/dicom_parser.html#dicom_parser.header.Header
 [pydicom]: https://pydicom.github.io/
-[series]: https://dicom-parser.readthedocs.io/en/latest/modules/dicom_parser.html#dicom_parser.series.Series
-[the documentation]: http://dicom-parser.readthedocs.io/?badge=latest
+[the documentation]: http://dicom-parser.readthedocs.io/?badge=latest
```

Comment:
Codacy has a fix for the issue: [Files should end with a single newline character](https://app.codacy.com/gh/ZviBaratz/dicom_parser/pullRequest?prid=7629233)
```suggestion
[the documentation]: http://dicom-parser.readthedocs.io/?badge=latest

```